matrices.tex

```diff
@@ -25,6 +25,8 @@
 {x - sqrt(y*(x + y)): 1, x + sqrt(y*(x + y)): 1}
 \end{verbatim}
 
+%% TODO a Python list is not a dense representation...
+%% Is this supposed to say flat? - scopatz
```

Comment:
I like the footnote, but I would have to read the whole paper through again to see if that section is still in my L1 or L2 by the time I get to this section.  Since the "makes it" wording is still in there, I think that a comment tying this back would make this more explicit, "dense (in the same sense as a "dense polynomial")"., or make that a footnote.